backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_privileges.py

```diff
@@ -129,6 +132,106 @@ def test_board_admin_can_deactivate_privilege(self):
 
         self._assert_the_privilege_was_deactivated()
 
+    @patch('cc_common.config._Config.current_standard_datetime', datetime.fromisoformat('2024-11-08T23:59:59+00:00'))
+    @patch('cc_common.config._Config.email_service_client')
+    def test_deactivate_privilege_handler_sends_expected_email_notifications(self, mock_email_service_client):
+        """
+        If a board admin has admin permission in the privilege jurisdiction, they can deactivate a privilege
+        """
+        self._load_provider_data()
+
+        # The user has admin permission for ne
+        resp = self._request_deactivation_with_scopes('openid email ne/aslp.admin')
+        self.assertEqual(200, resp['statusCode'])
+        self.assertEqual({'message': 'OK'}, json.loads(resp['body']))
+
+        mock_email_service_client.send_provider_privilege_deactivation_email.assert_called_once_with(
+            compact='aslp',
+            provider_email='björkRegisteredEmail@example.com',
+            privilege_id='SLP-NE-1',
+        )
+        mock_email_service_client.send_jurisdiction_privilege_deactivation_email.assert_called_once_with(
+            compact='aslp',
+            jurisdiction='ne',
+            privilege_id='SLP-NE-1',
+            provider_first_name='Björk',
+            provider_last_name='Guðmundsdóttir',
+        )
+
+    @patch('cc_common.config._Config.current_standard_datetime', datetime.fromisoformat('2024-11-08T23:59:59+00:00'))
+    def test_invalid_request_exception_raised_if_privilege_already_deactivated(self):
+        """
+        If a board admin does not have admin permission in the privilege jurisdiction, they cannot deactivate a
+        privilege.
+        """
+        self._load_provider_data()
+        with open('../common/tests/resources/dynamo/privilege.json') as f:
+            privilege = json.load(f)
+            # set persisted status to deactivated
+            privilege['persistedStatus'] = 'inactive'
+            self.config.provider_table.put_item(Item=privilege)
+
+        # The user has admin permission for oh
+        resp = self._request_deactivation_with_scopes('openid email ne/aslp.admin')
+        self.assertEqual(400, resp['statusCode'])
+        self.assertEqual({'message': 'Privilege already deactivated'}, json.loads(resp['body']))
+
+    @patch('cc_common.config._Config.current_standard_datetime', datetime.fromisoformat('2024-11-08T23:59:59+00:00'))
+    @patch('cc_common.config._Config.email_service_client')
+    def test_deactivate_privilege_handler_still_sends_state_notification_if_provider_notification_failed_to_send(
```

Comment:
Should we stick to jurisdiction for consistency?

```suggestion
    def test_deactivate_privilege_handler_still_sends_jurisdiction_notification_if_provider_notification_failed_to_send(
```